tests/functional/modules/test_zos_operator_func.py

```diff
@@ -141,3 +183,37 @@ def test_response_come_back_complete(ansible_zos_module):
         # HASP646 Only appears in the last line that before did not appears
         last_line = len(stdout)
         assert "HASP646" in stdout[last_line - 1]
+
+
+def test_zos_operator_parallel_terminal(get_config):
+    if zoau_version_checker.is_zoau_version_higher_than("1.3.2.0"):
+        path = get_config
+        with open(path, 'r') as file:
+            enviroment = yaml.safe_load(file)
+        ssh_key = enviroment["ssh_key"]
+        hosts = enviroment["host"].upper()
+        user = enviroment["user"].upper()
+        python_path = enviroment["python_path"]
+        cut_python_path = python_path[:python_path.find('/bin')].strip()
+        zoau = enviroment["environment"]["ZOAU_ROOT"]
+        try:
+            playbook = "playbook.yml"
+            inventory = "inventory.yml"
+            os.system("echo {0} > {1}".format(quote(PARALLEL_RUNNING.format(
+                zoau,
+                cut_python_path,
+            )), playbook))
+            os.system("echo {0} > {1}".format(quote(INVENTORY.format(
+                hosts,
+                ssh_key,
+                user,
+                python_path,
+            )), inventory))
+            command = "(ansible-playbook -i {0} {1}) & (ansible-playbook -i {0} {1})".format(inventory, playbook)
+            stdout = subprocess.check_output(command, shell=True, text=True)
+            assert "OMVS0001" in stdout
```

Comment:
Can you share the playbook output? I'm worried to have hardcoded stuff, if OMVS comes from the user OMVSADM then might be better to change this now, but if it refers to the OMVS segment it might be ok

Reply:
```
ok: [zvm] => {
    "output": {
        "changed": true,
        "cmd": "d a,all",
        "content": [
            "EC01130A   2024197  14:22:47.00             ISF031I CONSOLE OMVS0001 ACTIVATED",
            "EC01130A   2024197  14:22:47.00            -D A,ALL ",
            "EC01130A   2024197  14:22:47.00             CNZ4106I 14.22.43 DISPLAY ACTIVITY 630",
            "                                            JOBS     M/S    TS USERS    SYSAS    INITS   ACTIVE/MAX VTAM     OAS",
            "                                           00013    00014    00001      00034    00037    00001/00020       00023",
            "                                            *MASTER* *MASTER*          NSW  *   A=0001   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=005.019S  ET=19.21.30",
            "                                                                                WUID=STC00002 USERID=+MASTER+",
            "                                                                                WKL=SYSTEM   SCL=SYSTEM   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            PCAUTH   PCAUTH            NSW  *   A=0002   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=19.21.30",
            "                                                                                WKL=SYSTEM   SCL=SYSTEM   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            RASP     RASP              NSW  *   A=0003   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=19.21.30",
            "                                                                                WKL=SYSTEM   SCL=SYSTEM   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            TRACE    TRACE             NSW  *   A=0004   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=19.21.30",
            "                                                                                WKL=SYSTEM   SCL=SYSTEM   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            DUMPSRV  DUMPSRV  DUMPSRV  NSW  *   A=0005   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.010S  ET=19.21.28",
            "                                                                                WKL=SYSTEM   SCL=SYSTEM   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            XCFAS    XCFAS    IEFPROC  NSW  *   A=0006   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=011.304S  ET=19.21.28",
            "                                                                                WKL=SYSTEM   SCL=SYSTEM   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            GRS      GRS               NSW  *   A=0007   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=001.374S  ET=19.21.30",
            "                                                                                WKL=SYSTEM   SCL=SYSTEM   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            SMSPDSE  SMSPDSE           NSW  *   A=0008   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.866S  ET=19.21.30",
            "                                                                                WKL=SYSTEM   SCL=SYSTEM   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            CONSOLE  CONSOLE           NSW  *   A=0009   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=001.244S  ET=19.21.30",
            "                                                                                WKL=SYSTEM   SCL=SYSTEM   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            WLM      WLM      IEFPROC  NSW  *   A=000A   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=021.598S  ET=19.21.28",
            "                                                                                WKL=SYSTEM   SCL=SYSTEM   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            ANTMAIN  ANTMAIN  IEFPROC  NSW  *   A=000B   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.282S  ET=19.21.28",
            "                                                                                WKL=SYSTEM   SCL=SYSTEM   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            ANTAS000 ANTAS000 IEFPROC  NSW  *   A=000C   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.022S  ET=19.21.26",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            DEVMAN   DEVMAN   IEFPROC  NSW  *   A=000D   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.019S  ET=19.21.28",
            "                                                                                WKL=SYSTEM   SCL=SYSTEM   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            GTZ      GTZ      GTZ      NSW  *   A=000E   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.005S  ET=19.21.26",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            OMVS     OMVS     OMVS     NSW  *   A=000F   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=004.820S  ET=19.21.28",
            "                                                                                WKL=SYSTEM   SCL=SYSTEM   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            IEFSCHAS IEFSCHAS          NSW  *   A=0011   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=19.21.30",
            "                                                                                WKL=SYSTEM   SCL=SYSTEM   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            JESXCF   JESXCF   IEFPROC  NSW  *   A=0012   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.321S  ET=19.21.28",
            "                                                                                WKL=SYSTEM   SCL=SYSTEM   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            ALLOCAS  ALLOCAS           NSW  *   A=0013   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.002S  ET=19.21.30",
            "                                                                                WKL=SYSTEM   SCL=SYSTEM   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            IOSAS    IOSAS    IEFPROC  NSW  *   A=0015   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.707S  ET=19.21.28",
            "                                                                                WKL=SYSTEM   SCL=SYSTEM   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            IXGLOGR  IXGLOGR  IEFPROC  NSW  *   A=0014   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.172S  ET=19.21.28",
            "                                                                                WKL=SYSTEM   SCL=SYSTEM   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            AXR      AXR      IEFPROC  NSW  *   A=0016   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.019S  ET=19.21.26",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            CEA      CEA      IEFPROC  NSW  *O  A=0017   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.026S  ET=19.21.28",
            "                                                                                WKL=SYSTEM   SCL=SYSTEM   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            SMF      SMF      IEFPROC  NSW  *   A=0019   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.073S  ET=19.21.28",
            "                                                                                WKL=SYSTEM   SCL=SYSTEM   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            SMS      SMS      IEFPROC  NSW  *   A=0023   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=002.300S  ET=19.21.26",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            LLA      LLA      LLA      NSW  S   A=001B   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.188S  ET=19.21.26",
            "                                                                                WKL=SYSTEM   SCL=SYSTEM   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            VLF      VLF      VLF      NSW  S   A=001E   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.121S  ET=19.21.26",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            SDSF     SDSF     SDSF     NSW  S   A=0022   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=001.285S  ET=19.20.12",
            "                                                                                WUID=STC00004 USERID=OMVSKERN",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            CATALOG  CATALOG  IEFPROC  NSW  *   A=0024   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=001.417S  ET=19.21.25",
            "                                                                                WKL=SYSTEM   SCL=SYSTEM   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            ZFS      ZFS      ZFZGO    NSW  *   A=0025   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=027.064S  ET=19.21.22",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            RESOLVER RESOLVER IEFPROC  NSW  *O  A=0010   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.031S  ET=19.21.13",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            BPXOINIT BPXOINIT BPXOINIT OWT  *O  A=001C   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.128S  ET=19.21.13",
            "                                                                                WKL=SYSTEM   SCL=SYSTEM   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            JES2     JES2     IEFPROC  NSW  S   A=0020   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=012.685S  ET=19.20.59",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            JES2AUX  JES2AUX           NSW  *   A=0026   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=19.20.59",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            JES2MON  JES2MON  IEFPROC  NSW  *   A=0027   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=009.816S  ET=19.20.59",
            "                                                                                WKL=SYSTEM   SCL=SYSTEM   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=0028   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00007 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            RMF      RMF      IEFPROC  NSW  S   A=002B   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=003.653S  ET=19.20.12",
            "                                                                                WUID=STC00008 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=002D   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00011 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=002E   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00014 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=002F   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.001S  ET=NOTAVAIL",
            "                                                                                WUID=STC00015 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=0030   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00017 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=0031   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00019 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=0032   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00020 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=0033   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00018 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            SDSFAUX  SDSFAUX  SDSFAUX  NSW  SO  A=002A   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=014.160S  ET=19.20.11",
            "                                                                                WUID=STC00028 USERID=OMVSKERN",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=0029   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00035 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=001F   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00034 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=002C   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00037 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=0021   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00036 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=0035   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00039 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=0036   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00038 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=0037   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00041 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=0038   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00040 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=0039   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00044 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=003A   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00042 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=003B   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00043 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=003C   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00046 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=003D   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00045 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=003E   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00048 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=003F   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00047 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=0040   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00049 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            BPXAS    BPXAS    IEFPROC  OWT  IO  A=0041   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.008S  ET=19.18.37",
            "                                                                                WUID=STC00051 USERID=OMVSKERN",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            IGVDGNPP IGVDGNPP IGVDGNPP OWT  S   A=0042   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.003S  ET=19.20.09",
            "                                                                                WUID=STC00053 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            IMSVTAM  IMSVTAM  IEFPROC  NSW  S   A=0046   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=001.219S  ET=19.19.41",
            "                                                                                WUID=STC00061 USERID=SYSDSP",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            TSO      TSO      STEP1    NSW  S   A=0044   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.006S  ET=19.19.02",
            "                                                                                WUID=STC00063 USERID=USRT001",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            TNF      TNF      IEFPROC  NSW  *   A=0047   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.004S  ET=19.18.54",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            VMCF     VMCF     IEFPROC  NSW  *   A=0048   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.036S  ET=19.18.54",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            TCPIP    TCPIP    TCPIP    NSW  SO  A=0045   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=007.159S  ET=19.18.52",
            "                                                                                WUID=STC00068 USERID=OMVSKERN",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            IZUANG1  IZUANG1  STEP1    NSW  *O  A=0018   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.008S  ET=19.18.39",
            "                                                                                WUID=STC00069 USERID=IZUSVR",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            IZUSVR1  IZUSVR1  ZOSMF    IN   *O  A=0049   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=131.576S  ET=19.18.39",
            "                                                                                WUID=STC00070 USERID=IZUSVR",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            DHCPCLNT DHCPCLNT DHCPCLNT OWT  SO  A=001A   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.013S  ET=19.18.39",
            "                                                                                WUID=STC00072 USERID=OMVSKERN",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INETD    INETD    *OMVSEX  OWT  SO  A=004B   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.011S  ET=19.18.39",
            "                                                                                WUID=STC00073 USERID=OMVSKERN",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            FTPD1    STEP1    OMVSKERN OWT  AO  A=0041   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.008S  ET=19.18.37",
            "                                                                                WUID=STC00051 USERID=OMVSKERN",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            TN3270   TN3270   TN3270   NSW  SO  A=004E   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=005.480S  ET=19.18.14",
            "                                                                                WUID=STC00077 USERID=OMVSKERN",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            ICSF     ICSF              NSW  S   A=004F   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.321S  ET=19.18.11",
            "                                                                                WUID=STC00080 USERID=USRT001",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            BPXAS    BPXAS    IEFPROC  OWT  IO  A=0053   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.453S  ET=19.17.56",
            "                                                                                WUID=STC00087 USERID=OMVSKERN",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            SSHD4    STEP1    OMVSKERN OWT  AO  A=0053   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.453S  ET=19.17.56",
            "                                                                                WUID=STC00087 USERID=OMVSKERN",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            BPXAS    BPXAS    IEFPROC  OWT  IO  A=005C   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.390S  ET=028.567S",
            "                                                                                WUID=STC00166 USERID=OMVSKERN",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            BPXAS    BPXAS    IEFPROC  OWT  IO  A=0050   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.003S  ET=020.841S",
            "                                                                                WUID=STC00167 USERID=OMVSADM",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            BPXAS    BPXAS    IEFPROC  OWT  IO  A=0052   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.242S  ET=020.736S",
            "                                                                                WUID=STC00168 USERID=OMVSADM",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            SSHD9    *OMVSEX  OMVSKERN OWT  AO  A=005C   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.390S  ET=028.567S",
            "                                                                                WUID=STC00166 USERID=OMVSKERN",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            BPXAS    BPXAS    IEFPROC  OWT  IO  A=005D   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.003S  ET=020.951S",
            "                                                                                WUID=STC00169 USERID=OMVSADM",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            BPXAS    BPXAS    IEFPROC  OWT  IO  A=0043   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.004S  ET=012.053S",
            "                                                                                WUID=STC00170 USERID=OMVSADM",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            BPXAS    BPXAS    IEFPROC  OWT  IO  A=0059   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.003S  ET=020.961S",
            "                                                                                WUID=STC00171 USERID=OMVSADM",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            BPXAS    BPXAS    IEFPROC  OWT  IO  A=005B   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.004S  ET=008.494S",
            "                                                                                WUID=STC00172 USERID=OMVSADM",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            BPXAS    BPXAS    IEFPROC  OWT  IO  A=0055   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.250S  ET=020.722S",
            "                                                                                WUID=STC00173 USERID=OMVSADM",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            BPXAS    BPXAS    IEFPROC  OWT  IO  A=0054   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.003S  ET=020.850S",
            "                                                                                WUID=STC00174 USERID=OMVSADM",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            OMVSADM  STEP1    OMVSADM  OWT  AO  A=0059   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.003S  ET=020.961S",
            "                                                                                WUID=STC00171 USERID=OMVSADM",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            OMVSADM1 STEP1    OMVSADM  OWT  AO  A=005D   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.003S  ET=020.951S",
            "                                                                                WUID=STC00169 USERID=OMVSADM",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            OMVSADM  STEP1    OMVSADM  OWT  AO  A=0054   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.003S  ET=020.850S",
            "                                                                                WUID=STC00174 USERID=OMVSADM",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            OMVSADM2 STEP1    OMVSADM  OWT  AO  A=0050   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.003S  ET=020.841S",
            "                                                                                WUID=STC00167 USERID=OMVSADM",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            OMVSADM3 STEP1    OMVSADM  OWT  AO  A=0052   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.242S  ET=020.736S",
            "                                                                                WUID=STC00168 USERID=OMVSADM",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            OMVSADM2 STEP1    OMVSADM  OWT  AO  A=0055   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.250S  ET=020.722S",
            "                                                                                WUID=STC00173 USERID=OMVSADM",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            OMVSADM3 STEP1    OMVSADM  OWT  AO  A=0043   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.004S  ET=012.053S",
            "                                                                                WUID=STC00170 USERID=OMVSADM",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            OMVSADM4 STEP1    OMVSADM  OWT  AO  A=005B   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.004S  ET=008.494S",
            "                                                                                WUID=STC00172 USERID=OMVSADM",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            BPXAS    BPXAS    IEFPROC  IN   IO  A=0060   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.009S  ET=008.239S",
            "                                                                                WUID=STC00175 USERID=OMVSADM",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            OMVSADM5 STEP1    OMVSADM  IN   AO  A=0060   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.009S  ET=008.239S",
            "                                                                                WUID=STC00175 USERID=OMVSADM",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            BPXAS    BPXAS    IEFPROC  IN   IO  A=005A   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.009S  ET=000.645S",
            "                                                                                WUID=STC00176 USERID=OMVSADM",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            OMVSADM4 STEP1    OMVSADM  IN   AO  A=005A   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.009S  ET=000.645S",
            "                                                                                WUID=STC00176 USERID=OMVSADM",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                           USRT001  OWT     A=005E   PER=NO   SMC=000  PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                            CT=000.076S  ET=16.40.13",
            "                                                            WUID=TSU00112",
            "                                                            WKL=TSOUSERS SCL=ALLTSO   P=1",
            "                                                            RGP=N/A      SRVR=NO  QSC=NO",
            "BGYSC0804I Using timeout of 300 centiseconds.",
            "BGYSC0801I CONSBUFPGNUM=128 - Console command output buffer memory size to be allocated: 131072 bytes.",
            "BGYSC0806I Using console name of OMVS0000.",
            "BGYSC0805I Using CART of ZOAU8822.",
            "BGYSC0807I Unable to activate console. Retrying with console name of OMVS0001.",
            "BGYSC0802I Console command returned output string size of 36233 bytes."
        ],
        "elapsed": 15.43,
        "failed": false,
        "rc": 0,
        "wait_time_s": 3
    }
}
```

Reply:
```
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.026S  ET=19.21.28",
            "                                                                                WKL=SYSTEM   SCL=SYSTEM   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            SMF      SMF      IEFPROC  NSW  *   A=0019   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.073S  ET=19.21.28",
            "                                                                                WKL=SYSTEM   SCL=SYSTEM   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            SMS      SMS      IEFPROC  NSW  *   A=0023   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=002.300S  ET=19.21.26",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            LLA      LLA      LLA      NSW  S   A=001B   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.188S  ET=19.21.26",
            "                                                                                WKL=SYSTEM   SCL=SYSTEM   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            VLF      VLF      VLF      NSW  S   A=001E   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.121S  ET=19.21.26",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            SDSF     SDSF     SDSF     NSW  S   A=0022   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=001.285S  ET=19.20.12",
            "                                                                                WUID=STC00004 USERID=OMVSKERN",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            CATALOG  CATALOG  IEFPROC  NSW  *   A=0024   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=001.417S  ET=19.21.25",
            "                                                                                WKL=SYSTEM   SCL=SYSTEM   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            ZFS      ZFS      ZFZGO    NSW  *   A=0025   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=027.064S  ET=19.21.21",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            RESOLVER RESOLVER IEFPROC  NSW  *O  A=0010   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.031S  ET=19.21.13",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            BPXOINIT BPXOINIT BPXOINIT OWT  *O  A=001C   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.128S  ET=19.21.13",
            "                                                                                WKL=SYSTEM   SCL=SYSTEM   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            JES2     JES2     IEFPROC  NSW  S   A=0020   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=012.685S  ET=19.20.59",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            JES2AUX  JES2AUX           NSW  *   A=0026   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=19.20.59",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            JES2MON  JES2MON  IEFPROC  NSW  *   A=0027   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=009.816S  ET=19.20.58",
            "                                                                                WKL=SYSTEM   SCL=SYSTEM   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=0028   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00007 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            RMF      RMF      IEFPROC  NSW  S   A=002B   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=003.653S  ET=19.20.12",
            "                                                                                WUID=STC00008 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=002D   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00011 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=002E   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00014 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=002F   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.001S  ET=NOTAVAIL",
            "                                                                                WUID=STC00015 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=0030   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00017 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=0031   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00019 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=0032   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00020 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=0033   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00018 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            SDSFAUX  SDSFAUX  SDSFAUX  NSW  SO  A=002A   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=014.160S  ET=19.20.11",
            "                                                                                WUID=STC00028 USERID=OMVSKERN",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=0029   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00035 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=001F   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00034 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=002C   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00037 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=0021   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00036 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=0035   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00039 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=0036   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00038 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=0037   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00041 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=0038   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00040 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=0039   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00044 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=003A   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00042 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=003B   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00043 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=003C   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00046 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=003D   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00045 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=003E   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00048 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=003F   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00047 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INIT     INIT     IEFPROC  OWT  I   A=0040   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.000S  ET=NOTAVAIL",
            "                                                                                WUID=STC00049 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            BPXAS    BPXAS    IEFPROC  OWT  IO  A=0041   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.008S  ET=19.18.37",
            "                                                                                WUID=STC00051 USERID=OMVSKERN",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            IGVDGNPP IGVDGNPP IGVDGNPP OWT  S   A=0042   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.003S  ET=19.20.08",
            "                                                                                WUID=STC00053 USERID=++++++++",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            IMSVTAM  IMSVTAM  IEFPROC  NSW  S   A=0046   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=001.219S  ET=19.19.40",
            "                                                                                WUID=STC00061 USERID=SYSDSP",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            TSO      TSO      STEP1    NSW  S   A=0044   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.006S  ET=19.19.02",
            "                                                                                WUID=STC00063 USERID=USRT001",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            TNF      TNF      IEFPROC  NSW  *   A=0047   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.004S  ET=19.18.54",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            VMCF     VMCF     IEFPROC  NSW  *   A=0048   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.036S  ET=19.18.54",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            TCPIP    TCPIP    TCPIP    NSW  SO  A=0045   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=007.159S  ET=19.18.51",
            "                                                                                WUID=STC00068 USERID=OMVSKERN",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            IZUANG1  IZUANG1  STEP1    NSW  *O  A=0018   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.008S  ET=19.18.39",
            "                                                                                WUID=STC00069 USERID=IZUSVR",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            IZUSVR1  IZUSVR1  ZOSMF    IN   *O  A=0049   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=131.576S  ET=19.18.39",
            "                                                                                WUID=STC00070 USERID=IZUSVR",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            DHCPCLNT DHCPCLNT DHCPCLNT OWT  SO  A=001A   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.013S  ET=19.18.38",
            "                                                                                WUID=STC00072 USERID=OMVSKERN",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            INETD    INETD    *OMVSEX  OWT  SO  A=004B   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.011S  ET=19.18.38",
            "                                                                                WUID=STC00073 USERID=OMVSKERN",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            FTPD1    STEP1    OMVSKERN OWT  AO  A=0041   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.008S  ET=19.18.37",
            "                                                                                WUID=STC00051 USERID=OMVSKERN",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            TN3270   TN3270   TN3270   NSW  SO  A=004E   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=005.480S  ET=19.18.14",
            "                                                                                WUID=STC00077 USERID=OMVSKERN",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            ICSF     ICSF              NSW  S   A=004F   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.321S  ET=19.18.10",
            "                                                                                WUID=STC00080 USERID=USRT001",
            "                                                                                WKL=SYSTEM   SCL=SYSSTC   P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            BPXAS    BPXAS    IEFPROC  OWT  IO  A=0053   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.453S  ET=19.17.56",
            "                                                                                WUID=STC00087 USERID=OMVSKERN",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            SSHD4    STEP1    OMVSKERN OWT  AO  A=0053   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.453S  ET=19.17.56",
            "                                                                                WUID=STC00087 USERID=OMVSKERN",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            BPXAS    BPXAS    IEFPROC  OWT  IO  A=005C   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.390S  ET=028.393S",
            "                                                                                WUID=STC00166 USERID=OMVSKERN",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            BPXAS    BPXAS    IEFPROC  OWT  IO  A=0050   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.003S  ET=020.666S",
            "                                                                                WUID=STC00167 USERID=OMVSADM",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            BPXAS    BPXAS    IEFPROC  OWT  IO  A=0052   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.242S  ET=020.561S",
            "                                                                                WUID=STC00168 USERID=OMVSADM",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            SSHD9    *OMVSEX  OMVSKERN OWT  AO  A=005C   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.390S  ET=028.393S",
            "                                                                                WUID=STC00166 USERID=OMVSKERN",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            BPXAS    BPXAS    IEFPROC  OWT  IO  A=005D   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.003S  ET=020.777S",
            "                                                                                WUID=STC00169 USERID=OMVSADM",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            BPXAS    BPXAS    IEFPROC  OWT  IO  A=0043   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.004S  ET=011.878S",
            "                                                                                WUID=STC00170 USERID=OMVSADM",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            BPXAS    BPXAS    IEFPROC  OWT  IO  A=0059   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.003S  ET=020.786S",
            "                                                                                WUID=STC00171 USERID=OMVSADM",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            BPXAS    BPXAS    IEFPROC  OWT  IO  A=005B   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.004S  ET=008.320S",
            "                                                                                WUID=STC00172 USERID=OMVSADM",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            BPXAS    BPXAS    IEFPROC  OWT  IO  A=0055   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.250S  ET=020.547S",
            "                                                                                WUID=STC00173 USERID=OMVSADM",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            BPXAS    BPXAS    IEFPROC  OWT  IO  A=0054   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.003S  ET=020.676S",
            "                                                                                WUID=STC00174 USERID=OMVSADM",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            OMVSADM  STEP1    OMVSADM  OWT  AO  A=0059   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.003S  ET=020.786S",
            "                                                                                WUID=STC00171 USERID=OMVSADM",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            OMVSADM1 STEP1    OMVSADM  OWT  AO  A=005D   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.003S  ET=020.777S",
            "                                                                                WUID=STC00169 USERID=OMVSADM",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            OMVSADM  STEP1    OMVSADM  OWT  AO  A=0054   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.003S  ET=020.676S",
            "                                                                                WUID=STC00174 USERID=OMVSADM",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            OMVSADM2 STEP1    OMVSADM  OWT  AO  A=0050   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.003S  ET=020.666S",
            "                                                                                WUID=STC00167 USERID=OMVSADM",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            OMVSADM3 STEP1    OMVSADM  OWT  AO  A=0052   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.242S  ET=020.561S",
            "                                                                                WUID=STC00168 USERID=OMVSADM",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            OMVSADM2 STEP1    OMVSADM  OWT  AO  A=0055   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.250S  ET=020.547S",
            "                                                                                WUID=STC00173 USERID=OMVSADM",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            OMVSADM3 STEP1    OMVSADM  OWT  AO  A=0043   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.004S  ET=011.878S",
            "                                                                                WUID=STC00170 USERID=OMVSADM",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            OMVSADM4 STEP1    OMVSADM  OWT  AO  A=005B   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.004S  ET=008.320S",
            "                                                                                WUID=STC00172 USERID=OMVSADM",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            BPXAS    BPXAS    IEFPROC  IN   IO  A=0060   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.009S  ET=008.065S",
            "                                                                                WUID=STC00175 USERID=OMVSADM",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            OMVSADM5 STEP1    OMVSADM  IN   AO  A=0060   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.009S  ET=008.065S",
            "                                                                                WUID=STC00175 USERID=OMVSADM",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            BPXAS    BPXAS    IEFPROC  IN   IO  A=005A   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.008S  ET=000.470S",
            "                                                                                WUID=STC00176 USERID=OMVSADM",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                            OMVSADM4 STEP1    OMVSADM  IN   AO  A=005A   PER=NO   SMC=000",
            "                                                                                PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                                                CT=000.008S  ET=000.470S",
            "                                                                                WUID=STC00176 USERID=OMVSADM",
            "                                                                                WKL=SYSTEM   SCL=SYSOTHER P=1",
            "                                                                                RGP=N/A      SRVR=NO  QSC=NO",
            "                                           USRT001  OWT     A=005E   PER=NO   SMC=000  PGN=N/A  DMN=N/A  AFF=NONE",
            "                                                            CT=000.076S  ET=16.40.13",
            "                                                            WUID=TSU00112",
            "                                                            WKL=TSOUSERS SCL=ALLTSO   P=1",
            "                                                            RGP=N/A      SRVR=NO  QSC=NO",
            "BGYSC0804I Using timeout of 300 centiseconds.",
            "BGYSC0801I CONSBUFPGNUM=128 - Console command output buffer memory size to be allocated: 131072 bytes.",
            "BGYSC0806I Using console name of OMVS0000.",
            "BGYSC0805I Using CART of ZOAU8822.",
            "BGYSC0802I Console command returned output string size of 36233 bytes."
        ],
        "elapsed": 11.69,
        "failed": false,
        "rc": 0,
        "wait_time_s": 3
    }
}
```